core/runtime/src/main/java/io/quarkiverse/operatorsdk/runtime/CRDConfiguration.java

```diff
@@ -46,4 +46,11 @@ public class CRDConfiguration {
      */
     @ConfigItem(defaultValue = DEFAULT_OUTPUT_DIRECTORY)
     public String outputDirectory;
+
+    /**
+     * Whether the extension should also generate CRDs for external (i.e. not tied to a reconciler) Custom Resources
+     */
+    @ConfigItem(defaultValue = "false")
+    public Boolean alsoGenerateExternal;
```

Comment:
Isn't *external* a bit confusing ? CRDS not used by reconcilers may still be internal, and used as dependent resources. I would simply call it `generateAllCrds`.

Reply:
Done, renamed to `generateAll` since CRD is implied by the context and that would adding CRDs would make the property name awkward in `application.properties`.